underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -554,9 +554,11 @@ private ObjectPermissions getPermissionsInternal() {
 
         bucketMode = S3AUtils.translateBucketAcl(acl, owner.getId());
         if (mUfsConf.isSet(PropertyKey.UNDERFS_S3_OWNER_ID_TO_USERNAME_MAPPING)) {
+          // Here accountOwner can be null if there is no mapping set for this owner id
```

Comment:
the javadoc for `CommonUtils.getValueFromStaticMapping` seems to disagree: `@return the mapped value if the key exists, otherwise returns ""`. can you fix one or the other?